lib/smack/SmackRep.cpp

```diff
@@ -154,8 +210,8 @@ void SmackRep::collectRegions(llvm::Module &M) {
 
 const Stmt* SmackRep::alloca(llvm::AllocaInst& i) {  
   const Expr* size = 
-    Expr::fn("$mul",lit(storageSize(i.getAllocatedType())),lit(i.getArraySize()));
-                       
+    Expr::fn(opName("$mul", {TRUNC(getIntSize(i.getOperand(0)))}), lit(storageSize(i.getAllocatedType())),lit(i.getArraySize()));
```

Comment:
After more thought, this should return a `size_t`-sized integer. So no `TRUNC`ation.